### PR TITLE
Update Kafka to 0.9.0.1 and bump project version

### PR DESCRIPTION
**Pull request by ijuma**

### CODE REVIEW

**Comment by ijuma** [comment]
Review by @miguno or @enothereska

File: docs/intro.rst [hunk 1]
```diff
@@ -184,7 +184,7 @@ Requirements
 ------------
 
 - Hadoop: Camus works with both MRv1 and YARN. We recommend CDH 5.3.x or HDP 2.2.x.
-- Kafka: 0.9.0.0-cp1
+- Kafka: 0.9.0.1 (0.9.0.1-cp1 recommended)
 - Schema Registry: Confluent Schema Registry 2.0.0
```
Comment (miguno) [inline docs/intro.rst]: We should also document that the schema registry version was bumped to 2.0.1.

> - Schema Registry: Confluent Schema Registry 2.0.1

Comment (ijuma) [inline docs/intro.rst]: Done.

**Comment by miguno** [comment]
LGTM

**Pull request merged**